grunttasks/json2html.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+
+// Four steps are performed to lint the files:
+//  1. Clean the directories
+//  2. Generate the JSON files from the PO files
+//  3. The JSON files are then written out to temporary HTML files
+//  4. The HTML files are linted
+
+module.exports = function (grunt) {
+  grunt.registerMultiTask('l10n-json-to-html', 'Convert l10n JSON files to HTML', function () {
+    var files = 0;
+    this.files.forEach(function (file) {
+      var content = '';
+      var src = file.src[0];
+      var pathname = src.split('/');
+      var locale = grunt.file.readJSON(src);
+      for (var val in locale) {
+        var value = locale[val];
+        if (typeof value !== 'object' && value !== '' && isNaN(value)) {
```

Comment:
Do we ever expect *anything* other than strings? If so, we may have deeper problems. What happens if a value is not an object, or hits true on `isNaN`, does that mean it's unchecked? I think we should count that as an error to be safe.